content/en/docs/kitex/Tutorials/advanced-feature/multi_service.md

```diff
@@ -144,11 +144,13 @@ if the server receives requests in the following cases:
 ## FAQ
 ### 1. What's the difference between Multi-Service and Combine Service?
 - Combine Service (A service formed by merging multiple services into one unified service by generating code with -combine-service flag)
+  - Code for one unified service is generated.
```

Comment:
It's not precise. code for all services (both combineservice and each service being combined) are generated.

Reply:
You're very right.

Reply:
@felix021 Fixed.